docs/content/latest/troubleshoot/cluster/replace_failed_peers.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Fix 2/3 failed peers
+linkTitle: Fix 2/3 failed peers
+description: Fix 2/3 failed peers
+aliases:
+  - /troubleshoot/cluster/fix-2-3-failed-peers/
+  - /latest/troubleshoot/cluster/fix-2-3-failed-peers/
+menu:
+  latest:
+    parent: troubleshoot-cluster
+    weight: 835
+isTocNested: true
+showAsideToc: true
+---
+
+When a RAFT peer fails, YugabyteDB executes an automatic remote bootstrap to create a new peer from the remaining ones.
+If 2/3 of RAFT peers fail for a given tablet, we have to manually execute the equivalent of a remote bootstrap. We
+ can get a list of tablets in `yb-master-ip:7000/tablet-replication` yb-admin gui. 
+
+
+Assuming we have a cluster where:
+
+- a given tablet with UUID `TABLET1`
+- 3 tablet peers, 1 in good working order, referred to as `NODE_GOOD` and two broken peers, referred as `NODE_BAD1` and `NODE_BAD2`
+- We will be copying some tablet related data from the good peer to each of the bad peers
+
+These are the steps to follow in such scenario:
+
+- stop the bad TS, say `NODE_BAD1`, as we will be changing file system data underneath
```

Comment:
come to think of it, we should move this stop to 
- after copying the data from good to bad1
- but before we remove the bad data

to minimize downtime (as archiving + copying might take a while)

Reply:
changed the steps